docs/api/wrapper/setValue.md

```diff
@@ -12,17 +12,25 @@ Sets value of a text-control input or select element and updates `v-model` bound
 import { mount } from '@vue/test-utils'
 import Foo from './Foo.vue'
 
-const wrapper = mount(Foo)
+test('setValue demo', async () => {
+  const wrapper = mount(Foo)
 
-const textInput = wrapper.find('input[type="text"]')
-textInput.setValue('some value')
+  const textInput = wrapper.find('input[type="text"]')
+  await textInput.setValue('some value')
 
-const select = wrapper.find('select')
-select.setValue('option value')
+  expect(wrapper.find('input[type="text"]').element.value).toBe('some value')
 
-// requires <select multiple>
-const multiselect = wrapper.find('select')
-multiselect.setValue(['value1', 'value3'])
+  const select = wrapper.find('select')
+  await select.setValue('option value')
+
+  expect(wrapper.find('select').element.value).toBe('option value')
+
+  // requires <select multiple>
+  const multiselect = wrapper.find('select')
+  await multiselect.setValue(['value1', 'value3'])
+
+  expect(wrapper.find('select').element.value).toBe(['value1', 'value3'])
```

Comment:
Actually, is this the way the value is returned in case of a multiselect?

Reply:
Good question! I'll take a look and see what that line looks like.

Reply:
` expect(wrapper.find('select').element.value).toBe('value1')`? Looks like only the first value is returned? That's kind of lame :laughing:

Reply:
Well, that goes to show how often I use multiple selects 😆

Haha, I've been trying to get this to work in `v1.0.3` but only just realized that change was only just released as `v1.0.4` 😴

Looking at the PR, the test performs the assertions as can be seen here: https://github.com/vuejs/vue-test-utils/pull/1554/files#diff-21606123e32f708f8a6962091664fb13R76 but that seems off to me as well. What is `select.element.selectedOptions`? I don't see that being used anywhere else?

Alternatively, you can also write assertions against the prop bound to the `v-model` directive.

Reply:
Good find! [selectedOptions](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/selectedOptions) should work really well for an example. I can add that example to the commit you added to the fork to prevent a rebase nightmare :sweat_smile:

Reply:
added in 3801438406ef80c710e6cb8cfb0a426a0b1b9445